lib/image_processor.js

```diff
@@ -43,6 +43,10 @@ module.exports = class ImageProcessor{
             var x = parseInt(data[3], 10) - 1;
             var y = parseInt(data[4], 10) - 1;
 
+            if(width === 0 && height === 0) {
```

Comment:
it may make sense to remove the comment at line 51 :thinking:

Reply:
You are right. Done!